server/sessions/bjiKiGSsjT1pHnh0IJYrT0ZLnxxAUBqQ.json

```diff
@@ -0,0 +1 @@
+{"cookie":{"originalMaxAge":null,"expires":null,"httpOnly":true,"path":"/","sameSite":true},"cartId":1,"__lastAccess":1591979261903,"cartItemId":8}
```

Comment:
Your `server/sessions` should be in `.gitignore`. We should not check in session files to `git`.

Reply:
Added it to `.gitignore`